### PR TITLE
[CS] Code Style fixes for Superfluous Whitespace

### PR DESCRIPTION
**Pull request by photodude**
Pull Request for Issue found Superfluous Whitespace.

### Summary of Changes
- Whitespace found at end of line
- Functions must not contain multiple empty lines in a row; found 2 empty lines

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style testing on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

### CODE REVIEW

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e8b132788c5bba34ed1efd86205d25201be0b3b9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19161">issues.joomla.org/tracker/joomla-cms/19161</a>.</sub>

File: administrator/components/com_joomlaupdate/restore.php [hunk 1]
```diff
@@ -958,7 +958,6 @@ public function isIgnoredDirectory($shortFilename)
 							}
 						}
 
-
```
Comment (wilsonge) [inline administrator/components/com_joomlaupdate/restore.php]: As usual not this file please :)

Comment (photodude) [reply administrator/components/com_joomlaupdate/restore.php]: We need to add this file/component to the exclusions list.

Comment (photodude) [reply administrator/components/com_joomlaupdate/restore.php]: I'm not great with this kind of fix in git, I hope I correctly reset the change right.
```sh
git checkout ae6800a  joomla-cms/administrator/components/com_joomlaupdate/restore.php
```

**Comment by zero-24** [comment]
Merged thanks

File: administrator/components/com_joomlaupdate/restore.php (hunk 1 above)
Comment (wilsonge) [reply administrator/components/com_joomlaupdate/restore.php]: I thought we had added it to exclusions :/ https://github.com/joomla/coding-standards/pull/214/files#diff-66782cea1e46c4c36349de42d1603b98R33

**Comment by photodude** [comment]
Thanks @zero-24

**Pull request merged**